test/tstDetailsTreeConstruction.cpp

```diff
@@ -63,16 +63,17 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(morton_codes, DeviceType, ARBORX_DEVICE_TYPES)
     details::expand(boxes_host(i), points[i]);
   Kokkos::deep_copy(boxes, boxes_host);
 
+  typename DeviceType::execution_space space{};
   ArborX::Box scene_host;
-  details::TreeConstruction<DeviceType>::calculateBoundingBoxOfTheScene(
-      boxes, scene_host);
+  ArborX::Details::TreeConstruction::calculateBoundingBoxOfTheScene(
```

Comment:
It looks a bit inconsistent to use `ArborX::Details` here and `details` in the lines above.

Reply:
Dropped namespace alias